[2023-12-20T11:49:54.808+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2023-12-20T11:49:54.841+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2022-03-10T00:00:00+00:00 [queued]>
[2023-12-20T11:49:54.841+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2023-12-20T11:49:54.874+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_id> on 2022-03-10 00:00:00+00:00
[2023-12-20T11:49:54.882+0000] {standard_task_runner.py:60} INFO - Started process 5972 to run task
[2023-12-20T11:49:54.899+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_id', 'scheduled__2022-03-10T00:00:00+00:00', '--job-id', '1427', '--raw', '--subdir', 'DAGS_FOLDER/first_dat.py', '--cfg-path', '/tmp/tmpwp0fk2sz']
[2023-12-20T11:49:54.903+0000] {standard_task_runner.py:88} INFO - Job 1427: Subtask second_id
[2023-12-20T11:49:55.095+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.second_id scheduled__2022-03-10T00:00:00+00:00 [running]> on host 29b3723064ac
[2023-12-20T11:49:55.332+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abrham' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_id' AIRFLOW_CTX_EXECUTION_DATE='2022-03-10T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2022-03-10T00:00:00+00:00'
[2023-12-20T11:49:55.334+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-20T11:49:55.335+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be runing after task 3']
[2023-12-20T11:49:55.373+0000] {subprocess.py:86} INFO - Output:
[2023-12-20T11:49:55.374+0000] {subprocess.py:93} INFO - hey, I am task2 and will be runing after task 3
[2023-12-20T11:49:55.375+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-20T11:49:55.441+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_id, execution_date=20220310T000000, start_date=20231220T114954, end_date=20231220T114955
[2023-12-20T11:49:55.514+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-20T11:49:55.596+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
